paper/paper.md

```diff
@@ -34,14 +49,23 @@ At present, it contained rules-based methods primarily designed for English.
 Several of the implementations are sourced from the Python NLTK project ([@NLTK1,@NLTK2]);
 although these were in turn source from older pre-existing methods.
 These include the very commonly used Penn Treebank Tokenizer ([@penntok]),
-and the multilingual Tok-tok ([@toktok], [@toktokpub]).
+the multilingual Tok-tok ([@toktok], [@toktokpub]) and tweet tokenizer([@tweettok]).
 Interesting, as an implementation detail, to implement many tokenizers,
 it was found most convenient to first port them to sed,
 and then use Julia's meta-programming capacity to generate Julia expressions from the sed code at compile time.
```

Comment:
We only have `penn_tokenizer` and `improved_penn_tokenizer`, using sedbased approach. How should I change this?